tests/unit/lifecycle/LifecycleTask.spec.js

```diff
@@ -82,9 +82,16 @@ describe('lifecycle task helper methods', () => {
 
             const res1 = lct._filterRules(mBucketRules, item1, objTags);
             assert.strictEqual(res1.length, 3);
-            const expRes1 = getRuleIDs(mBucketRules.filter(rule =>
-                (rule.Filter.Prefix && rule.Filter.Prefix.startsWith('atask'))
-                || !rule.Filter.Prefix));
+            const expRes1 = getRuleIDs(mBucketRules.filter(rule => {
+                if (!rule.Filter.Prefix) {
+                    return true;
+                }
+                if (rule.Filter.Prefix &&
```

Comment:
Now you can remove this first check, since test at https://github.com/scality/backbeat/pull/230/files#diff-70184adf6b658fa3260d32a93bb664dbR86 guarantees it already.